documentation/puppet_conf_setting_diffs.markdown

```diff
@@ -211,3 +211,28 @@ Puppet Server does not currently consider this setting for any code running on t
 ### [`http_proxy_port`](https://docs.puppetlabs.com/references/latest/configuration.html#httpproxyport)
 
 Puppet Server does not currently consider this setting for any code running on the master and using the `Puppet::Network::HttpPool` module to create an HTTP client connection. This pertains, for example, to any requests that the master would make to the `reporturl` for the `http` report processor. Note that Puppet agents do still honor this setting.
+
+## Overriding Puppet settings in Puppet Server
+
+Currently, the [`jruby-puppet` section of your `puppetserver.conf` file](configuration.markdown#puppetserver.conf) contains five settings
+(`master-conf-dir`, `master-code-dir`, `master-var-dir`, `master-run-dir`, and `master-log-dir`) that allow you to override settings set in
+your `puppet.conf` file. On installation, these five settings will be set to the default value that would normally be read by your
+`puppet.conf` file.
```

Comment:
I think I'd just say "will be set to the proper default values." If you want, you can point to the "dirs" pages like http://docs.puppetlabs.com/puppet/4.2/reference/dirs_codedir.html.